frontend/src/actions/users.js

```diff
@@ -179,15 +179,15 @@ function receiveUserDetail(userId, user) {
   };
 }
 
-export function addFriend(userId) {
+export function follow(userId) {
   return dispatch => {
-    return apiFetch('friend/' + userId, { method: 'PUT' })
+    return apiFetch('follow/' + userId, { method: 'PUT' })
       .then(response => response.json())
       .then(json => {
         if (json.success) {
-          toast.success('You are now Friends!');
+          toast.success('You are following User #' + userId);
```

Comment:
hmm let's not expose IDs in notifications as they don't really make sense

Reply:
I might get rid of that success toast then, because I already have the button changing the way it looks and I'm not sure what to use as a message that's not werid like 'You are a follower'. lol